httpcore/_async/http_proxy.py

```diff
@@ -132,7 +132,9 @@ async def _forward_request(
 
         if connection is None:
             connection = AsyncHTTPConnection(
-                origin=origin, http2=self._http2, ssl_context=self._ssl_context
+                origin=origin,
+                http2=self._http2,
+                ssl_context=self._ssl_context,
```

Comment:
I know I already mentioned it, but just in case — we need to get rid of these unrelated changes due to Black treating trailing commas a bit differently now.

The way to revert these would be to remove the trailing comma here and elsewhere…

```suggestion
                ssl_context=self._ssl_context
```

And then apply Black again so that it reformats things as before. :-)

Reply:
Already on it.

Reply:
Wait, am I still doing something wrong?

Reply:
Ok, done.